src/components/calculator.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  // constructor(props) {
+  //   super(props);
+  // }
```

Comment:
- [ ] Kindly make sure you remove all the commented code lines to make your codebase cleaner and  for best practices